docs/Components/BaseComponent.md

```diff
@@ -63,3 +63,10 @@ print(math.degrees(freebody_component.angle_of_attack))
 print(math.degrees(attached_component.angle_of_attack))
 >>> 4.0
 ```
+
+## empty_mass property
+This property returns the total mass of the set of nested components
```

Comment:
It returns the total mass of the nested components (not a set, as set is a specific python data type), excluding payload components.